ftw/solr/upgrades/20191211175444_provide_precompiled_bundle/registry.xml

```diff
@@ -0,0 +1,13 @@
+<registry>
+    <records prefix="plone.bundles/ftw-solr-resources"
```

Comment:
Please also remove the individual resources
```xml
    <records prefix="plone.resources/ftwsolr_liveserch" interface="Products.CMFPlone.interfaces.IResourceRegistry" remove="True" />
    <records prefix="plone.resources/ftwsolr_facets" interface="Products.CMFPlone.interfaces.IResourceRegistry" />
    <records prefix="plone.resources/ftwsolr_jquery_history" interface="Products.CMFPlone.interfaces.IResourceRegistry" />
    <records prefix="plone.resources/ftwsolr_jquery_legazy_highlight" interface="Products.CMFPlone.interfaces.IResourceRegistry" />
    <records prefix="plone.resources/ftwsolr_search" interface="Products.CMFPlone.interfaces.IResourceRegistry" />

```

Reply:
Done